test/vol.c

```diff
@@ -42,137 +42,6 @@ static const char *FILENAME[] = {"vol_test_file", NULL};
 #define N_ELEMENTS 10
 #define NAME_LEN   512
 
-/* A VOL class struct to verify registering optional operations */
```

Comment:
Definitions for the dynamic registration VOL were moved upwards into h5test to allow the MT VL tests to test dynamic op registration in parallel. It would probably be better to move them into a new header file only used by the MT VL tests and vol.c, but I ran into some issues implementing that so I've left it like this for now.